projects/admin/src/app/record/detail-view/library-detail-view/library-detail-view.component.ts

```diff
@@ -0,0 +1,45 @@
+/*
+ * RERO ILS UI
+ * Copyright (C) 2019 RERO
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, version 3 of the License.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import { Component, OnInit } from '@angular/core';
+import { Observable } from 'rxjs';
+import { DetailRecord } from '@rero/ng-core/lib/record/detail/view/detail-record';
+
+@Component({
+  selector: 'admin-library-detail-view',
+  templateUrl: './library-detail-view.component.html',
+  styles: []
+})
+export class LibraryDetailViewComponent implements DetailRecord, OnInit {
+
+  /** Observable resolving record data */
+  record$: Observable<any>;
+
+  /** Resource type */
+  type: string;
+
+  /** Record */
+  record: any;
+
+  constructor() { }
+
+  ngOnInit() {
+    this.record$.subscribe(record => {
```

Comment:
As @jma said in a previous comment, use async pipe instead of this.

Reply:
done